Docker/README.md

```diff
@@ -43,18 +44,20 @@ curl http://127.0.0.1:8888/v1/chain/get_info
 ## Start both nodeos and walletd containers
```

Comment:
Please replace all `walletd` references with `keosd`.

Reply:
OK, I'll do that.